users/views.py

```diff
@@ -156,6 +162,9 @@ def get_context_data(self, *, _object_list: Any = None, **kwargs: Any) -> dict:
         context_data["is_create_user_form"] = False
         return context_data
 
+    def get_success_message(self, cleaned_data: Any) -> str:
+        return self.success_message.format(email=self.object.email, action="updated")
```

Comment:
Yea, please move actions to enum in constants

Reply:
I’m not sure that I’ve done what you requested, so please let me know, so I will change it if needed.